handler/src/test/java/io/netty/handler/ssl/JdkSslEngineTest.java

```diff
@@ -263,6 +267,40 @@ public String select(List<String> protocols) {
         }
     }
 
+    @Test
+    public void testEnablingAnAlreadyDisabledSslProtocol() throws Exception {
```

Comment:
@blucas to make the same test work with OpenSslEngine and the jdk one you can do and instanceof .